packages/access-api/test/access-authorize.test.js

```diff
@@ -95,6 +99,129 @@ describe('access/authorize', function () {
+    const { issuer, service, conn, mf } = ctx
+    const accountDID = 'did:mailto:dag.house:email'
+
+    const inv = await Access.authorize
+      .invoke({
+        issuer,
+        audience: service,
+        with: issuer.did(),
+        nb: {
+          as: accountDID,
+        },
+      })
+      .execute(conn)
+
+    // @todo - this only returns string when ENV==='test'. Remove that env-specific behavior
+    assert.ok(typeof inv === 'string', 'invocation result is a string')
+
+    const confirmEmailPostUrl = new URL(inv)
+    const confirmEmailPostResponse = await mf.dispatchFetch(
+      confirmEmailPostUrl,
+      { method: 'POST' }
+    )
+    assert.deepEqual(
+      confirmEmailPostResponse.status,
+      200,
+      'confirmEmailPostResponse status is 200'
+    )
+
+    const claim = Access.claim.invoke({
+      issuer,
+      audience: conn.id,
+      with: issuer.did(),
+    })
+    const claimResult = await claim.execute(conn)
+    assert.ok(
+      'delegations' in claimResult,
+      'claimResult should have delegations property'
+    )
+    const claimedDelegations = Object.values(claimResult.delegations).flatMap(
+      (bytes) => {
+        return bytesToDelegations(
+          /** @type {import('@web3-storage/access/src/types.js').BytesDelegation} */ (
+            bytes
+          )
+        )
+      }
+    )
+    assert.deepEqual(
+      claimedDelegations.length,
+      2,
+      'should have claimed delegation(s)'
+    )
+    /**
+     * narrow Ucanto.Proof to Ucanto.Delegation
+     *
+     * @param {import('@ucanto/interface').Proof} proof
+     */
+    // eslint-disable-next-line unicorn/consistent-function-scoping
+    const proofDelegation = (proof) => {
+      if (!('cid' in proof)) {
+        throw new Error('proof must be delegation')
+      }
+      return proof
+    }
+    const attest = claimedDelegations.find(
+      (d) => proofDelegation(d.proofs[0])?.issuer.did() === accountDID
+    )
+    assert.ok(
+      attest,
+      'should claim ucan/attest delegation with proof.iss=accountDID'
+    )
+    assert.deepEqual(attest.issuer.did(), service.did())
+    assert.deepEqual(attest.audience.did(), issuer.did())
+    assert.deepEqual(attest.capabilities[0].can, 'ucan/attest')
+    assert.deepEqual(attest.capabilities[0].with, service.did())
+
+    // ucan/attest nb.proof can be decoded into a delegation
+    const claimedNb = attest.capabilities[0].nb
+    assert.ok(
+      claimedNb && typeof claimedNb === 'object' && 'proof' in claimedNb,
+      'should have nb.proof'
+    )
+    const expectAccountToKey = proofDelegation(attest.proofs[0])
+    assert.ok(expectAccountToKey, 'expect proofs to contain delegation')
+    assert.deepEqual(expectAccountToKey.issuer.did(), accountDID)
+    assert.deepEqual(expectAccountToKey.audience.did(), issuer.did())
+    assert.deepEqual(expectAccountToKey.capabilities, [
```

Comment:
remove this whole delegation if it's not needed for this test

Reply:
done in https://github.com/web3-storage/w3protocol/pull/460/commits/685af659cfe1c7f6ab09e79a36cce36df3aa0ecb